src/sentry/static/sentry/app/components/forms/selectField.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import FormField from './formField';
+import StyledSelect from './select.styled';
+
+export default class SelectField extends FormField {
+  static propTypes = {
+    ...FormField.propTypes,
+    options: PropTypes.array.isRequired,
+    onChange: PropTypes.func,
+    clearable: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    ...FormField.defaultProps,
+    clearable: true,
+  };
+
+  getClassName() {
+    return '';
+  }
+
+  onChange = opt => {
+    const value = opt ? opt.value : null;
+    this.setValue(value);
+  };
+
+  renderArrow = () => {
+    return <span className="icon-arrow-down" />;
```

Comment:
I think we should use `InlineSvg` here. If you are struggling with width or stroke issues, try something like:

```jsx
styled(props => <InlineSvg src="icon-arrow-down" {...props>)`
  stroke-width: //different stroke width
  width: //different svg width
`
```

And if that still doesn't work I would be happy to make you a new SVG, but I really want to avoid adding in legacy icon font stuff.

Reply:
@Chrissy  A new SVG would be good, the shape of this one is different, even with the stroke width it doesn't look the same.

Reply:
It matches the arrow in the other dropdowns, we should update them all together